lightstep/sdk/metric/internal/viewstate/base_instrument.go

```diff
@@ -96,22 +96,40 @@ func (metric *instrumentBase[N, Storage, Auxiliary, Methods]) mergeDescription(d
 	}
 }
 
-func attributeFilter(viewf *attribute.Filter) attribute.Filter {
-	return func(kv attribute.KeyValue) bool {
-		if kv.Key == "" {
+// isValidAttribute supports filtering invalid attributes.  Note, this
+// should be fast, trye not to allocate!  TODO: match the
+// specification.
```

Comment:
Can you link the spec here?

Reply:
Done. The spec is somewhat ambiguous. :shrug: